server/src/main/java/io/deephaven/server/table/TableServiceGrpcImpl.java

```diff
@@ -280,6 +280,7 @@ public void batch(final BatchTableRequest request,
             final StreamObserver<ExportedTableCreationResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
             final SessionState session = sessionService.getCurrentSession();
+            session.getAuthContext().requirePrivilege(TableServicePrivilege.CAN_BATCH);
```

Comment:
I'm assuming that each of the individual operations in a batch still need privs, correct?